src/main/java/uk/ac/ebi/eva/pipeline/jobs/VariantAggregatedConfiguration.java

```diff
@@ -21,59 +21,81 @@
 import org.springframework.batch.core.Step;
 import org.springframework.batch.core.configuration.annotation.EnableBatchProcessing;
 import org.springframework.batch.core.configuration.annotation.JobBuilderFactory;
+import org.springframework.batch.core.job.builder.FlowJobBuilder;
 import org.springframework.batch.core.job.builder.JobBuilder;
+import org.springframework.batch.core.job.flow.Flow;
 import org.springframework.batch.core.launch.support.RunIdIncrementer;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Import;
 
+import org.springframework.context.annotation.Scope;
 import uk.ac.ebi.eva.pipeline.jobs.steps.VariantsLoad;
 import uk.ac.ebi.eva.pipeline.jobs.steps.VariantsTransform;
 
 /**
- * Job similar to VariantConfiguration, but it takes an aggregated VCF,
- * which has no sample information (e.g. no genotypes).
+ *  Complete pipeline workflow for aggregated Vcf.
+ *  Aggregated statistics are provided in the VCF instead of the genotypes.
  *
- * 1. transform the VCF into an intermediate json file
- * 2. load it into mongo.
+ *  transform ---> load --> (optionalVariantAnnotationFlow: variantsAnnotGenerateInput --> (annotationCreate --> variantAnnotLoad))
+ *
+ *  Steps in () are optional
  */
 @Configuration
 @EnableBatchProcessing
-@Import({VariantsLoad.class, VariantsTransform.class})
+@Import({VariantsLoad.class, VariantAnnotConfiguration.class})
 public class VariantAggregatedConfiguration extends CommonJobStepInitialization{
 
     private static final Logger logger = LoggerFactory.getLogger(VariantAggregatedConfiguration.class);
+
     private static final String jobName = "load-aggregated-vcf";
-    private static final String NORMALIZE_VARIANTS = "Normalize variants";
-    private static final String LOAD_VARIANTS = "Load variants";
+    public static final String NORMALIZE_VARIANTS = "Normalize variants";
+    public static final String LOAD_VARIANTS = "Load variants";
+
+    //Job default values
+    {
+        includeSamples = false;
+        compressGenotypes = false;
+        calculateStats = true;
+        includeStats = true;
+    }
```

Comment:
Wasn't it agreed to encapsulate these fields into small POJOs that would be injected from a job-specific `@Bean` method? Did that cause any issues?